package.json

```diff
@@ -33,6 +34,7 @@
  "scripts": {
   "build": "grunt build && grunt create-build-zip",
   "deploy": "grunt deploy",
-  "dev": "cross-env BABEL_ENV=default webpack --watch"
+  "dev": "cross-env BABEL_ENV=default webpack --watch",
+  "lint": "eslint ."
```

Comment:
Should this also include a call to `phpcs`?

Reply:
We could do `lint:js`, `lint:php`, and `lint`, which calls the former two, if that makes sense?

Reply:
Yeah, that sounds good. In another PHP project I have a `composer.json` with:

```
  "scripts": {
    "lint": [
      "@php-l",
      "phpcs"
    ],
    "php-l": "git ls-files -z -- '*.php' | xargs -0 -n 1 -P 4 php -l",
    "phpcs": "phpcs",
    "phpcbf": "phpcbf",
    "test": "WP_TESTS_DIR=/app/public/core-dev/tests/phpunit/ phpunit --config=phpunit-integration.xml.dist",
    "generate-readme": "@php ./vendor/xwp/wp-dev-lib/generate-markdown-readme"
  }
```

Reply:
This begs the question whether we wanna put all scripts in `package.json` or `composer.json`. Or keep them separated, i.e. PHP things in `composer.json` and JS things in `package.json` 🤷‍♂️

Reply:
Done now in the most recent commit.